bumble/host.py

```diff
@@ -139,6 +139,7 @@ def __init__(
         super().__init__()
 
         self.hci_metadata = {}
+        self.controller_init_required = False
```

Comment:
It's not clear what the purpose of this new property is that couldn't be done with the `ready` property already. For drivers that have to send commands and receive responses before a reset has been done, they should set the `host.ready` property to `True` to allow incoming packets to be processed. Drivers are responsible for leaving the controller in an initialized and reset state when returning from `driver.init_controller()`, with `host.ready` therefore set to `True` as well.

Reply:
Ok sound good I'll do the appropriate modifications then